modules/nodes-descheduler-configuring-profiles.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * nodes/scheduling/nodes-descheduler.adoc
+
+[id="nodes-descheduler-configuring-profiles_{context}"]
+= Configuring descheduler profiles
+
+You can configure which profiles the descheduler uses to evict pods.
+
+.Prerequisites
+
+* Cluster administrator privileges
+
+.Procedure
+
+. Edit the `KubeDescheduler` object:
++
+[source,terminal]
+----
+$ oc edit kubedeschedulers.operator.openshift.io cluster -n openshift-kube-descheduler-operator
+----
+
+. Specify one or more profiles in the `spec.profiles` section.
++
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1beta1
+kind: KubeDescheduler
+metadata:
+  name: cluster
+  namespace: openshift-kube-descheduler-operator
+spec:
+  deschedulingIntervalSeconds: 3600
+  logLevel: Normal
+  managementState: Managed
+  operatorLogLevel: Normal
+  profiles:
+  - AffinityAndTaints       <1>
+  - TopologyAndDuplicates   <2>
+  - LifecycleAndUtilization <3>
+----
+<1> Enable the `AffinityAndTaints` profile, which evicts pods that violate basic pod and node affinity and node taints.
```

Comment:
@bergerhoffer should we rephrase the sentence here ? which evicts pods that violate basic pod node affinity & node taints ?

Reply:
@kasturinarra I was trying to capture the 3 different strategies. I think I've reworded it better here, let me know if this looks better to you:

```
... which evicts pods that voilate inter-pod anti-affinity, node affinity, and node taints.
```